model/project.go

```diff
@@ -32,46 +32,38 @@ const (
-	DeactivatePrevious bool `yaml:"deactivate_previous" bson:"deactivate_previous,omitempty"`
+	Enabled             bool                       `yaml:"enabled,omitempty" bson:"enabled"`
+	Stepback            bool                       `yaml:"stepback,omitempty" bson:"stepback"`
+	PreErrorFailsTask   bool                       `yaml:"pre_error_fails_task,omitempty" bson:"pre_error_fails_task,omitempty"`
+	PostErrorFailsTask  bool                       `yaml:"post_error_fails_task,omitempty" bson:"post_error_fails_task,omitempty"`
+	OomTracker          bool                       `yaml:"oom_tracker,omitempty" bson:"oom_tracker"`
+	BatchTime           int                        `yaml:"batchtime,omitempty" bson:"batch_time"`
+	Owner               string                     `yaml:"owner,omitempty" bson:"owner_name"`
+	Repo                string                     `yaml:"repo,omitempty" bson:"repo_name"`
+	RemotePath          string                     `yaml:"remote_path,omitempty" bson:"remote_path"`
+	Branch              string                     `yaml:"branch,omitempty" bson:"branch_name"`
+	Identifier          string                     `yaml:"identifier,omitempty" bson:"identifier"`
+	DisplayName         string                     `yaml:"display_name,omitempty" bson:"display_name"`
+	CommandType         string                     `yaml:"command_type,omitempty" bson:"command_type"`
+	Ignore              []string                   `yaml:"ignore,omitempty" bson:"ignore"`
+	Parameters          []ParameterInfo            `yaml:"parameters,omitempty" bson:"parameters,omitempty"`
+	Pre                 *YAMLCommandSet            `yaml:"pre,omitempty" bson:"pre"`
+	Post                *YAMLCommandSet            `yaml:"post,omitempty" bson:"post"`
+	Timeout             *YAMLCommandSet            `yaml:"timeout,omitempty" bson:"timeout"`
+	EarlyTermination    *YAMLCommandSet            `yaml:"early_termination,omitempty" bson:"early_termination,omitempty"`
+	CallbackTimeout     int                        `yaml:"callback_timeout_secs,omitempty" bson:"callback_timeout_secs"`
+	Modules             ModuleList                 `yaml:"modules,omitempty" bson:"modules"`
+	BuildVariants       BuildVariants              `yaml:"buildvariants,omitempty" bson:"build_variants"`
+	Functions           map[string]*YAMLCommandSet `yaml:"functions,omitempty" bson:"functions"`
+	TaskGroups          []TaskGroup                `yaml:"task_groups,omitempty" bson:"task_groups"`
+	Tasks               []ProjectTask              `yaml:"tasks,omitempty" bson:"tasks"`
+	ExecTimeoutSecs     int                        `yaml:"exec_timeout_secs,omitempty" bson:"exec_timeout_secs"`
+	Loggers             *LoggerConfig              `yaml:"loggers,omitempty" bson:"loggers,omitempty"`
+	CommitQueueAliases  []ProjectAlias             `yaml:"commit_queue_aliases,omitempty" bson:"commit_queue_aliases,omitempty"`
```

Comment:
This is going to end up conflicting with your other PR FYI since we remove 62 - 66 in there